### PR TITLE
 add test for export key --qr-html 

### PR DESCRIPTION
**Pull request by enkore**
Follow up to #2135

### CODE REVIEW

File: borg/paperkey.html [hunk 1]
```diff
@@ -2171,8 +2171,11 @@
         }
     }
 
-
-
+    /* center the QR code in the page */
```
Comment (ThomasWaldmann) [inline borg/paperkey.html]: center *on* the page?

File: borg/paperkey.html [hunk 2]
```diff
@@ -2217,7 +2220,7 @@
     <div id="printout">
         <div id="title" contenteditable>BorgBackup Printable Key Backup</div>
         <div contenteditable>To restore either scan the QR code below, decode it and import it using
-<pre>borg key-import /path/to/repo scannedfile</pre>
+<pre>borg key import /path/to/repo scannedfile</pre>
```
Comment (ThomasWaldmann) [inline borg/paperkey.html]: hmm, what is the format (mimetype) of scannedfile?

"scannedfile" sounds a bit like .tiff or other bitmap format, but I guess it needs to be text? or binary?

File: borg/testsuite/archiver.py [hunk 3]
```diff
@@ -1318,6 +1318,19 @@ def test_key_export_repokey(self):
...
+        repo_id = self._extract_repository_id(self.repository_path)
+        self.cmd('key', 'export', '--qr-html', self.repository_location, export_file)
+
+        with open(export_file, 'r') as fd:
+            export_contents = fd.read()
+
+        assert bin_to_hex(repo_id) in export_contents
+        assert export_contents.startswith('<!doctype html>')
+        assert export_contents.endswith('</html>')
+
```
Comment (ThomasWaldmann) [inline borg/testsuite/archiver.py]: good :)

File: borg/paperkey.html (hunk 2 above)
Comment (enkore) [inline borg/paperkey.html]: imho clear enough that with "scan the QR code" using a QR scanner is meant not an imaging scanner. In the former case you'll get text / text file which can be directly consumed here. (After copying to PC).

Comment (ThomasWaldmann) [inline borg/paperkey.html]: ok

**Comment by enkore** [comment]
=> #2149

**Pull request closed without merging**